wordpress-mobile/WordPress-iOS
# Improve media previews on long-press

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/22139

- The preview now works even if the image hasn’t been loaded yet (loads lazily)
- Increase the size of the preview container to match the `.large` image size
- Load `.large` images after briefly displaying the `.small` image
- Add missing `accessibilityIgnoresInvertColors` in `ImageView`
- Add support for previewing video (verify that is muted)
- Add support for previewing documents (not really, but it now at least shows something and you can see the full title)

To test:

- Verify based on the list of changes above

> [!IMPORTANT]  
> Make sure to test video on the physical device

## Regression Notes
1. Potential unintended areas of impact: Site Media
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
